tests/dataaccess/common/testdata/checkinputerrors.csv

```diff
@@ -0,0 +1,41 @@
+test,error
+nondictinput,Input must be a dictionary
```

Comment:
Why are these specified like this?   this says checkinputerrors?

Reply:
Checkinputerrors.csv loads in the dictionary of expected error messages to check against query error messages. These two messages are checked against each other to ensure that they match.